libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -37,6 +37,7 @@
 typedef struct OVOptions{
     char *device_type;
     int nireq;
+    int input_resizable;
```

Comment:
fix_input_resizable in git commit log.

Reply:
OK~

Reply:
please give a background in commit log, for example, the openvino model file needs a specific value for input while the model actually accepts different input size, etc.

Reply:
OK~

Reply:
input_resize in commit log, 'random' to 'different'

Reply:
OK

Reply:
i still see "adding input_resize option here for easier use.", it is input_resizable

Reply:
sorry, my mistake